src/components/cards/JufoExpertDetailCard.tsx

```diff
@@ -99,72 +99,83 @@ export const JufoExpertDetailCard: React.FC<Props> = (props) => {
-          <div className={classes.header}>
-            <Title size="h4" bold>
-              {props.expert.firstName} {props.expert.lastName}
-            </Title>
-
-            <div className={classes.rightHeader}>
-              {props.expert.projectFields.map((field) => (
-                <Tag
-                  key={`${field}-${props.expert.id}`}
-                  fontSize="12px"
-                  background="#F4F6FF"
-                  color="#4E6AE6"
-                >
-                  {field}
-                </Tag>
-              ))}
-              {user.isProjectCoachee && (
-                <Button
-                  backgroundColor="#4E6AE6"
-                  color="#ffffff"
-                  className={classes.emailButton}
-                  image={
-                    pinned ? (
-                      <Icons.BookmarkSlashFilled />
-                    ) : (
-                      <Icons.BookmarkFilled />
-                    )
-                  }
-                  onClick={pinned ? unpinExpert : pinExpert}
-                />
-              )}
-              <Button
-                backgroundColor="#4E6AE6"
-                color="#ffffff"
-                image={<Icons.EmailFilled />}
-                onClick={openEmailDialog}
-                className={classes.emailButton}
-              />
+    <>
+      <span
+        onClick={openExpertDetails}
+        className={classes.containerButton}
+        aria-hidden="true"
+      >
+        <div className={classes.container}>
+          <div className={classes.infoContainer}>
+            <div className={classes.header}>
+              <Title size="h4" bold>
+                {props.expert.firstName} {props.expert.lastName}
+              </Title>
+
+              <div className={classes.rightHeader}>
+                {props.expert.projectFields.map((field) => (
+                  <Tag
+                    key={`${field}-${props.expert.id}`}
+                    fontSize="12px"
+                    background="#F4F6FF"
+                    color="#4E6AE6"
+                  >
+                    {field}
+                  </Tag>
+                ))}
+                {user.isProjectCoachee && (
+                  // <Tooltip placement="topLeft" title="anpinnen">
+                  <span data-tooltip="anpinnen">
+                    <Button
+                      backgroundColor="#4E6AE6"
+                      color="#ffffff"
+                      className={classes.emailButton}
+                      image={
+                        pinned ? (
+                          <Icons.BookmarkSlashFilled />
+                        ) : (
+                          <Icons.BookmarkFilled />
+                        )
+                      }
+                      onClick={pinned ? unpinExpert : pinExpert}
+                    />
+                  </span>
+                )}
+                <span data-tooltip="Expert*in kontaktieren">
```

Comment:
Can you try to make this a little bit wider such that it fits into one line?
<img width="172" alt="Bildschirmfoto 2021-03-28 um 04 08 59" src="https://user-images.githubusercontent.com/33378936/112740210-61d48880-8f7b-11eb-874b-608578c2829a.png">